Examples/Tests/PML/analysis_pml_psatd_rz.py

```diff
@@ -0,0 +1,44 @@
+#! /usr/bin/env python
+
+# Copyright 2021 David Grote
+#
+#
+# This file is part of WarpX.
+#
+# License: BSD-3-Clause-LBNL
+
+"""
+This script tests the absorption of fields in the PML in RZ geometry.
+
+The input file inputs_particle_rz is used: it features an electron
+moving radially that launches a pulse. This scripts runs until
+most of the pulse escapes the radial boundary. If the PML fails,
+the pulse will remain with in the domain.
+"""
+import sys
+import numpy as np
+import yt
+yt.funcs.mylog.setLevel(0)
+sys.path.insert(1, '../../../../warpx/Regression/Checksum/')
+import checksumAPI
+
+# Open plotfile specified in command line
+filename = sys.argv[1]
+ds = yt.load( filename )
+
```

Comment:
I think we started introducing this in other analysis scripts. Maybe it is good to add it consistently to all scripts, especially the new ones, so that developers become familiar with it? Not 100% sure.
```suggestion

    # yt 4.0+ has rounding issues with our domain data:
    # RuntimeError: yt attempted to read outside the boundaries
    # of a non-periodic domain along dimension 0.
    if 'force_periodicity' in dir(ds): ds.force_periodicity()
    
```

Reply:
I'll add it.